test/DaprExtensionTests/Tests/DaprTopicTriggerTests.cs

```diff
@@ -75,6 +77,27 @@ public async Task TopicNameInAttributeAsBindingExpression()
             Assert.Contains(input.ToString(), functionLogs);
         }
 
+        [Fact]
+        public async Task TopicNameAndRouteAreDifferent()
+        {
+            int input = 42;
+
+            // The method name is DotNetMethodName
```

Comment:
This test looks the same as ExplicitTopicNameInAttribute

Reply:
I removed this and split up one of the other tests